### PR TITLE
PD-1126 Update IPv6 Configuration Content

### PR DESCRIPTION
**Pull request by micjohnson777**
This PR adds information and instruction on setting up IPv6 dual stacking using either the UI or Console Setup menu to the ConfigureIPv6.md article.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALE/SCALETutorials/Network/ConfigureIPv6.md [hunk 1]
```diff
@@ -51,6 +52,51 @@ Click on **Add** in the **Interfaces** to open the **Add Interface** screen.
...
+
+   Click **Settings** on the **Global Configuration** widget. The **Edit Global Configuration** screen opens.
+
+   Enter the IPv6 address for the gateway in **IPv6 Default Gateway**.
+   
+   Click **Save**.
+
+2. Add the IPv6 static IP address to the primary interface.
+
+   Select the primary interface **en8s0**, then click **Edit**.
```
Comment (DjP-iX) [inline content/SCALE/SCALETutorials/Network/ConfigureIPv6.md:80]:
```suggestion
   Select the primary interface, **en8s0**, then click **Edit**.
```

File: content/SCALE/SCALETutorials/Network/ConfigureIPv6.md [hunk 2]
```diff
@@ -51,6 +52,51 @@ Click on **Add** in the **Interfaces** to open the **Add Interface** screen.
...
+
+   Click **Add*** to the right of **Aliases** to add another set of IP Address fields.
+
+   Enter the IPv6 address, then select the netmask.
+
+   Click **Save**
+
+3. Test the network change.
+   To verify the IPv6 address, in a new browser window, enter the address inside square brackets. For example, *[ffff:ff:59f8:100::12].
```
Comment (DjP-iX) [inline content/SCALE/SCALETutorials/Network/ConfigureIPv6.md:89]: is this supposed to be italicized?


```suggestion
   To verify the IPv6 address, in a new browser window, enter the address inside square brackets. For example, *[ffff:ff:59f8:100::12]*.
```

File: content/SCALE/SCALETutorials/Network/ConfigureIPv6.md [hunk 3]
```diff
@@ -51,6 +52,51 @@ Click on **Add** in the **Interfaces** to open the **Add Interface** screen.
...
+
+   Log into the UI, and click **Save Changes**.
+
+   Log out of that browser session, return to your other UI session.
+   Both IPv4 and IPv6 addresses should show on the screen for the primary interface.
+
+After installing SCALE and using the Console Setup menu to configure system networking and set up dual stacking, add the name servers and both IP protocol default gateways in general network settings (option 2 on the menu), then add both IP address with netmasks as aliases on the primary network interface (option 1 on the menu).
```
Comment (DjP-iX) [inline content/SCALE/SCALETutorials/Network/ConfigureIPv6.md:96]:
```suggestion
After installing SCALE and using the Console Setup menu to configure system networking and set up dual stacking, add the name servers and both IP protocol default gateways in general network settings (option 2 on the menu), then add both IP address, with netmasks as aliases, on the primary network interface (option 1 on the menu).
```

**Review by DjP-iX — CHANGES_REQUESTED** [review]

File: content/SCALE/SCALETutorials/Network/ConfigureIPv6.md (hunk 2 above)
Comment (micjohnson777) [reply content/SCALE/SCALETutorials/Network/ConfigureIPv6.md:89]: yes in this instance this is an example or variable so italics is correct.

**Review by DjP-iX — APPROVED** [review]

**Pull request merged**